api/app/utils/daily_pdf_gen.py

```diff
@@ -0,0 +1,43 @@
+from typing import List, Mapping
+import pdfkit
+from jinja2 import Template
+
+from app.schemas.hfi_calc import FireCentre, HFIResultResponse, PlanningArea, WeatherStation
+from app.utils.pdf_data_formatter import response_2_daily_jinja_format
+
+daily_template_path = "api/app/utils/daily_template.html"
+daily_rendered_path = "api/app/utils/daily_rendered.html"
+
+output_file_path = "api/app/utils/out.pdf"
```

Comment:
Same as daily_rendered_path - this has to go into a temporary file. (Ideally in-memory)

Reply:
in memory now, as a of: https://github.com/bcgov/wps/pull/1763/commits/4e333fd132ab77cf21de524d5282b9bd8f573941